components/infra-proxy-service/server/policyfiles.go

```diff
@@ -51,7 +51,7 @@ func (s *Server) GetPolicyfile(ctx context.Context, req *request.Policyfile) (*r
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 
-	result, err := c.GetRoleList()
+	result, err := c.SearchRoles(&request.SearchQuery{})
```

Comment:
Keeping the this extended `SearchRoles` methods as it is, as I have called coupled of places, changing it will `c.client.Search.Query()` need to update with `SeachResult` so skipping for now.